templates/accounts/pending.html

```diff
@@ -43,16 +48,30 @@ <h2>Uploaded sounds awaiting moderation {% if moderators_version %} by  <a class
                     </div>
                 </div>
                 <div class="sample_information" style="width: 250px;">
+                    <span>
+                        {% if ticket.assignee %}
+                            Assigned to <a href="{% url "account" ticket.assignee.username %}">{{ ticket.assignee.username }}</a>
```

Comment:
Maybe say "Assigned to moderator {{uname}}" instead of "Assigned to {{uname}}". This small change might make things a bit more clear for some users.